ui/app/hooks/useRetryTransaction.js

```diff
@@ -34,7 +34,7 @@ export function useRetryTransaction(transactionGroup) {
 
       trackMetricsEvent();
       await dispatch(fetchBasicGasEstimates);
-      const transaction = initialTransaction;
+      const transaction = primaryTransaction;
```

Comment:
Just making sure I understand the impact of this change: it looks like this will ensure that when you retry a failed cancellation, it retries the _cancellation_ and not the original?